### PR TITLE
Add multipart functionality for oncall and add docker compose for tes…

### PR DESCRIPTION
**Pull request by martinhny**
…ting

### CODE REVIEW

File: pkg/sender/office365.go [hunk 1]
```diff
@@ -78,6 +78,10 @@ func mapToGraphMail(email *email.SkylineEmail) (*graphusers.ItemSendMailPostRequ
 		contentType := graphmodels.HTML_BODYTYPE
 		body.SetContentType(&contentType)
 		body.SetContent(&email.HTML)
+	} else if email.IsMultiPartAlternative() {
```
Comment (evenh) [inline pkg/sender/office365.go:81]: Could be merged with previous `if` block

File: local_testing/Dockerfile
Comment (evenh) [inline local_testing/Dockerfile:1]: Why a separate Dockerfile?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
+version: "3.9"
+
+x-environment: &oncall-environment
```
Comment (evenh) [inline docker-compose.yml:3]: Haven't seen this approach before, neat.

**Review by evenh — APPROVED** [review]

File: local_testing/Dockerfile
Comment (martinhny) [reply local_testing/Dockerfile:1]: The 'main' Dockerfile is used by goreleaser. I don't think theres a way to build an image with this Dockerfile without goreleaser, and I wanted docker compose to build the image for faster testing

File: pkg/sender/office365.go (hunk 1 above)
Comment (martinhny) [reply pkg/sender/office365.go]: true, fixed

File: local_testing/Dockerfile
Comment (evenh) [reply local_testing/Dockerfile:1]: There might be, but let's not waste any time on that right now. Go with the current approach!

**Review by evenh — APPROVED** [review]

**Review by berg-thom — APPROVED** [review]

**Pull request merged**